feature/registration-signin.md

```diff
@@ -0,0 +1,34 @@
+# Registration and Sign In
+
+Although Cambiatus uses EOS, we have a Sign Up/Sign In process that is a little different from other blockchain apps. First of all we try to abstract public and private key pairs from the user.
+
+## Basic registration steps
+
+If the user isn't trying to register from an Invitation, we use the basic registration process. It all begins by the user clicking on `Don't have an account yet? Create now` on the Login page. From there the user will be take to the registration page, where he/she will have to fill in 3 infos: Full name, account name (which is an [EOS account](integration/accounts.html)) and an email address. 
+
+The frontend will generate this request:
```

Comment:
Maybe it's better to say "this kind of request" since it will be different each time?